core/src/test/java/org/apache/iceberg/TableTestBase.java

```diff
@@ -307,6 +353,30 @@ ManifestFile writeDeleteManifest(int newFormatVersion, Long snapshotId, DeleteFi
     return writer.toManifestFile();
   }
 
+  InputFile writeManifestList(String compressionCodec, ManifestFile... manifestFiles)
+      throws IOException {
+    File manifestListFile = temp.newFile();
+    Assert.assertTrue(manifestListFile.delete());
+    OutputFile outputFile =
+        org.apache.iceberg.Files.localOutput(
+            FileFormat.AVRO.addExtension(manifestListFile.toString()));
+
+    try (FileAppender<ManifestFile> writer =
+        ManifestLists.write(
+            formatVersion,
+            outputFile,
+            SNAPSHOT_ID,
+            SNAPSHOT_ID - 1,
+            formatVersion > 1 ? 34L : 0,
+            compressionCodec,
+            /* compressionLevel */ null)) {
+      for (ManifestFile manifestFile : manifestFiles) {
+        writer.add(manifestFile);
+      }
+    }
+    return outputFile.toInputFile();
```

Comment:
Style: there should be an empty newline between a control flow block and the following statement.

Reply:
Added a blank line.